homeassistant/components/azure_data_explorer/client.py

```diff
@@ -0,0 +1,92 @@
+"""Setting up the ingest client."""
+from __future__ import annotations
+
+import io
+import logging
+
+from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+from azure.kusto.data.data_format import DataFormat
+from azure.kusto.ingest import (
+    IngestionProperties,
+    ManagedStreamingIngestClient,
+    QueuedIngestClient,
+    StreamDescriptor,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+# Suppress very verbose logging from client
+logger = logging.getLogger("azure")
+logger.setLevel(logging.WARNING)
+
+
+class AzureDataExplorerClient:
+    """Class for Azure Data Explorer Client."""
+
+    def __init__(
+        self,
+        CONF_ADX_CLUSTER_INGEST_URI: str,
+        CONF_ADX_DATABASE_NAME: str,
+        CONF_ADX_TABLE_NAME: str,
+        CONF_APP_REG_ID: str,
+        CONF_APP_REG_SECRET: str,
+        CONF_AUTHORITY_ID: str,
+        CONF_USE_FREE: bool,
```

Comment:
```suggestion
        CONF_ADX_CLUSTER_INGEST_URI: str,
        CONF_ADX_DATABASE_NAME: str,
        CONF_ADX_TABLE_NAME: str,
        CONF_APP_REG_ID: str,
        CONF_APP_REG_SECRET: str,
        CONF_AUTHORITY_ID: str,
        CONF_USE_FREE: bool,
```

```suggestion
        data: dict[str, Any]
```

Only constant should be written all upper case.

Reply:
resolved ny lowercasing parameters